buildkite/shared-build-env.dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM vimc/node-docker-openjdk:master
+FROM vimc/node-docker:master
+
+RUN echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/stretch-backports.list
+RUN apt-get update
+RUN apt-get install -t stretch-backports -y \
+    openjdk-11-jdk openjdk-8-jdk
```

Comment:
`openjdk-8-jdk` should be removed, this is added to stop certificate mismatch complain.